src/asset/index.ts

```diff
@@ -12,21 +13,19 @@ function getIdentifier(origin: string, id: string): string {
 }
 
 // eslint-disable-next-line max-len
-export function getKusamaAsset(assetId: string, balance: number|string|Balance, address: string, configuration: NetworkConfiguration): Asset {
-  const image = configuration.unit.image ? configuration.unit.image : "";
-  const customViewUrl = configuration.unit.customViewUrl ?
-    configuration.unit.customViewUrl : `https://polkascan.io/pre/kusama/account/${address}`;
-  return {
+export function getKusamaAsset(
```

Comment:
You might wanna rename this. getPolkadotAssetDescription.
AssetId could also go in config